src/coap_io.c

```diff
@@ -1248,6 +1248,52 @@ coap_network_read(coap_socket_t *sock, coap_packet_t *packet) {
 
 #if !defined(WITH_CONTIKI)
 
+unsigned int
```

Comment:
As COAP_EPOLL_SUPPORT should be undefined for Contiki anyway, compiling this function in Contiki should be no problem.

Reply:
Updated